engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/analyse/alias/graph/ScopeImpl.scala

```diff
@@ -80,7 +112,7 @@ sealed private[graph] class ScopeImpl(
         val newScope =
           new ScopeImpl(
             childScopeCopies.toList,
-            occurrences,
+            occurrencesById.asScala,
```

Comment:
That conversion is probably expensive, could potentially save some cycles if you are looking for some optimization

Reply:
- this code doesn't get executed during `Many*` benchmarks
- however this code may be slowing down #12806

> probably expensive

- is it?
- this is a conversion of `java.util.Map` to `mutable.Map`
- that one should be fast
- as it is just a simple wrapper around the original object, isn't it?